components/Templates/includes/functions-view_template.php

```diff
@@ -91,6 +91,12 @@ function frontier_decode_template( $code, $atts ) {
  */
 function frontier_if_block( $atts, $code ) {
 
+	$atts = wp_parse_args( $atts, array(
+		'pod'   => null,
+		'id'    => null,
+		'field' => null,
+	) );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 5e1614da718afc1ccfe1263d50b07be9</sub>